test/logger.spec.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import logger from '../src/logger';
+
+describe('Logger', () => {
+  it('info mode should not fail', () => {
+    expect(() => logger.info('Testing message')).to.not.throw(Error);
+  });
+
+  it('info mode should not fail', () => {
+    expect(() => logger.debug('Testing message')).to.not.throw(Error);
+  });
+
+  it('error mode should not fail', () => {
+    expect(() => logger.error('Testing message')).to.not.throw(Error);
+  });
+});
```

Comment:
repeated test cases 
add one test case where it throws

Reply:
The exceptions are tested in the library it self, you can check.
https://github.com/winstonjs/winston/blob/master/test/exception-handler.test.js